Dispatcher/Views/Base.lproj/Main.storyboard

```diff
@@ -146,28 +138,45 @@
                                 <color key="backgroundColor" name="backgroundColor"/>
                                 <constraints>
                                     <constraint firstItem="5xd-B7-bhd" firstAttribute="leading" secondItem="Wjn-wg-k7J" secondAttribute="trailing" id="Hcx-zy-6rr"/>
+                                    <constraint firstAttribute="height" relation="lessThanOrEqual" constant="95" id="T1u-xE-DPK"/>
                                     <constraint firstAttribute="trailing" secondItem="5xd-B7-bhd" secondAttribute="trailing" id="VVi-w3-l20"/>
+                                    <constraint firstAttribute="width" relation="greaterThanOrEqual" constant="393" id="aRU-Ly-YRt"/>
                                     <constraint firstItem="aaR-rJ-CiT" firstAttribute="leading" secondItem="9ji-6p-TrG" secondAttribute="leading" constant="10" id="eGF-mj-Vx8"/>
                                     <constraint firstAttribute="bottom" secondItem="Wjn-wg-k7J" secondAttribute="bottom" id="heu-tc-Ghm"/>
                                     <constraint firstAttribute="bottom" secondItem="aaR-rJ-CiT" secondAttribute="bottom" id="lWe-dE-lIY"/>
                                     <constraint firstAttribute="bottom" secondItem="5xd-B7-bhd" secondAttribute="bottom" id="par-Zd-U57"/>
-                                    <constraint firstAttribute="height" constant="95" id="tv6-aQ-fDW"/>
                                 </constraints>
                             </view>
+                            <tableView clipsSubviews="YES" contentMode="scaleAspectFill" alwaysBounceVertical="YES" showsHorizontalScrollIndicator="NO" showsVerticalScrollIndicator="NO" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="-1" estimatedSectionHeaderHeight="-1" sectionFooterHeight="-1" estimatedSectionFooterHeight="-1" translatesAutoresizingMaskIntoConstraints="NO" id="Ntx-aV-dth" userLabel="ArticlesTable View">
+                                <rect key="frame" x="0.0" y="152" width="393" height="700"/>
+                                <color key="backgroundColor" name="screenBackgroundColor"/>
+                                <constraints>
+                                    <constraint firstAttribute="height" constant="700" id="LFR-iF-fdQ"/>
```

Comment:
You can't give a fixed height to a tableView - it might look good on the device that you are building on, but it will be too big or too small on another device. Find different constraints to achieve the design.